tests/cli/test_externalshare.py

```diff
@@ -0,0 +1,96 @@
+import logging
+
+import pytest
+import responses
+from bson import ObjectId
+from click.testing import CliRunner
+from responses import matchers
+
+from drapps.externalshare import external_share
+
+logger = logging.getLogger()
+
+
+@pytest.mark.parametrize('app_id', [str(ObjectId())])
+@responses.activate
+@pytest.mark.parametrize('use_name', [True, False])
+@pytest.mark.parametrize('allow_external_sharing', [True, False])
+@pytest.mark.parametrize('users_to_add', [['new.user.1@datarobot.com', 'new.user.2@datarobot.com']])
+@pytest.mark.parametrize('users_to_remove', ['@enron.com'])
```

Comment:
Remember this repo is public, we shouldn't reference other companies even innocently 

```suggestion
@pytest.mark.parametrize('users_to_remove', ['@some-external-domain.com'])
```